sig-summit/README.md

```diff
@@ -5,7 +5,8 @@ This SIG is focused on planning and alignment for the annual Spinnaker Summit co
 ## Meetings
 
 * Regular SIG Meeting
-  * Every other Tuesday, 10 AM US/Pacific ([See in your time zone](https://www.thetimezoneconverter.com/?t=10am&tz=San%20Francisco))
+  * Every other Wednesday, 11 AM US/Pacific ([See in your time zone](https://www.thetimezoneconverter.com/?t=10am&tz=San%20Francisco))
```

Comment:
I think you might've forgotten to update the link here. It's still for 10 AM 😸

Reply:
Updated the timezone link to 11am pst